### ISSUE TITLE
Sharik should use KDE's native file manager.

### ISSUE THREAD

**Issue by akomoy** [marchellodev/sharik#62]
On Plasma it's annoying to get GTK's file manager so it'd be great if that can be changed.

**Comment by marchellodev** [comment]
Did you install the app as a snap?

**Comment by marchellodev** [comment]
I believe this issue should be fixed is Sharik v3. Can you please verify? https://github.com/marchellodev/sharik/issues/63

I'll close this issue for now

**Comment by akomoy** [comment]
Sorry for the late reply, just tried it out and it still uses GTK's file manager. I'm using the appimage version. I'll check the aur version when it's released. The dark mode is very good btw.

**Comment by marchellodev** [comment]
Yep, it seems like the issue still persist. Can you please check if this will solve it?

https://askubuntu.com/questions/1150404/kubuntu-18-10-how-do-i-change-this-file-picker

**Comment by marchellodev** [comment]
I've just run Sharik in kde neon, adding `GTK_USE_PORTAL=1` have fixed the issue.

![image](https://user-images.githubusercontent.com/61112084/124116874-25339400-da78-11eb-8116-e8f989bc7e70.png)

I guess the issue is solved now

**Comment by marchellodev** [comment]
@Ygarr Does this work for you?
